README.md

```diff
@@ -51,6 +51,23 @@ environment.
         }
      }
 
+### Collector Resource
+
+When send a POST request to create a collector, the request must have the below URI and include the following key/value pairs in the payload -- hostname, payload and payload['nonce']:
+
+*Request Example*
+
+    `POST /api/v1/collectors
+
+    body: {
+     hostname=www.ikea.com,
+     payload={ nonce: 'kjhs9182791hakajh19',
```

Comment:
Let's put the nonce at as a top level parameter instead of inside the payload. That way the payload can be created by the capture routine in the extension and passed through to the api without any changes.